cdk/cdk.ts

```diff
@@ -1,5 +1,9 @@
-import * as cdk from "@aws-cdk/core";
 import { PinBoardStack } from "./stack";
+import { App, DefaultStackSynthesizer } from "aws-cdk-lib";
 
-const app = new cdk.App();
-new PinBoardStack(app, "PinBoardStack");
+const app = new App();
+new PinBoardStack(app, "PinBoardStack", {
+  synthesizer: new DefaultStackSynthesizer({
+    generateBootstrapVersionRule: false,
+  }),
```

Comment:
Worth noting that GuCDK uses a `LegacyStackSynthesizer` here (see https://github.com/guardian/cdk/pull/1174/files#r847210569 for a little more info).

Any reason to not use GuCDK in this app?

Reply:
Oh:

> NOTE: we'll be adopting https://github.com/guardian/cdk in a future PR (soonish) as it wasn't mature when this CDK stack was first created.

Reply:
https://trello.com/c/OlfYW4Xf/804-adopt-gucdk-in-pinboard is the ticket to use GuCDK which is in this coming sprint.